TrafficCapture/dockerSolution/build.gradle

```diff
@@ -1,9 +1,10 @@
 plugins {
     id 'org.opensearch.migrations.java-library-conventions'
     id "com.avast.gradle.docker-compose" version "0.16.12"
-    id "com.bmuschko.docker-java-application" version "9.3.1"
+    id 'com.bmuschko.docker-remote-api'
```

Comment:
What is this required for?  I assume that the plugin is effectively being loaded by CommonUtils now.  Is this line required?  If so, can you document why and put a version on it?

Reply:
The id 'com.bmuschko.docker-remote-api' plugin is necessary for the _remaining_ DockerBuildImage tasks to function properly. While this plugin is now declared in the `buildSrc/build.gradle` file, it's not automatically applied to the `trafficCaptureProxyServerTest/build.gradle` file and needs to be explicitly declared. As for the version number, adding it here causes a conflict because the plugin is already on the classpath due to including it in the `buildSrc/build.gradle` file. So to sum it up, this line IS required but W/O a specified version. The version is managed within the `buildSrc/build.gradle` file.

Reply:
Thanks.  I understand the version.  Thanks for the reference.
For my own future understanding, the plugin isn't imported in the common code, just some java imports from it are and the remote-api plugin is a better fit to pickup the remaining other pieces unique for this grade file.